packages/widget/src/pages/SettingsPage/ResetSettingsButton.tsx

```diff
@@ -31,29 +50,67 @@ export const ResetSettingsButton: React.FC = () => {
         tools.bridges.map((tool) => tool.key),
         tools.exchanges.map((tool) => tool.key),
       );
+
+      console.log({ config });
```

Comment:
Please add ESLint and husky rules for this, it happens too often. 😄

Reply:
About time for this 🥲
Surely I'll do it